generators/client/index.js

```diff
@@ -82,10 +82,11 @@ module.exports = class extends BaseBlueprintGenerator {
                 }
             },
 
-            setupClientconsts() {
+            setupClientConstants() {
```

Comment:
This one is a breaking change. You should add it to https://github.com/jhipster/generator-jhipster/pull/10788 instead ;-)

Reply:
Ok I'll gladly revert and move these changes to the suggested PR.

These methods didn't appear to be used anywhere, hence I didn't think it'd be an issue.

Am I correct in understanding that you're saying it'll break blueprints and modules extending these core sub-generators?

Reply:
Thanks! Yes, you're correct: these methods could be used externally and thus should be considered as public API.

Reply:
Cool! 
I've moved the breaking changes to https://github.com/jhipster/generator-jhipster/pull/10788